services/simcore/docker-compose.yml

```diff
@@ -372,8 +455,8 @@ services:
         - traefik.http.routers.${PREFIX_STACK_NAME}_dask_scheduler.middlewares=${PREFIX_STACK_NAME}_dask_scheduler_replace_regex@docker, ops_gzip@docker, ops_auth@docker
       resources:
         limits:
-          memory: 256M
-          cpus: '1.000'
+          memory: 128M
+          cpus: '0.1'
```

Comment:
for this one. I remember the dask-scheduler was once killed because it went over 128MiB. I think @mrnicegyu11 might also remember. So I am wondering why this guy is now so much down-scaled?

Reply:
As discussed in whereby